CCT_cold/experiment.js

```diff
@@ -0,0 +1,665 @@
+  "<button type='button' class = select-button2 id = 5 onclick = chooseButton1(this.id)>5</button>"+
+  "<button type='button' class = select-button2 id = 6 onclick = chooseButton1(this.id)>6</button>"+
+  "<button type='button' class = select-button2 id = 7 onclick = chooseButton1(this.id)>7</button>"+
+  "<button type='button' class = select-button2 id = 8 onclick = chooseButton1(this.id)>8</button>"+
+  "<button type='button' class = select-button2 id = 9 onclick = chooseButton1(this.id)>9</button>"+
+  "<button type='button' class = select-button2 id = 10  onclick = chooseButton1(this.id)>10</button>"+
+  "<button type='button' class = select-button2 id = 11  onclick = chooseButton1(this.id)>11</button>"+
+  "<button type='button' class = select-button2 id = 12  onclick = chooseButton1(this.id)>12</button>"+
+  "<button type='button' class = select-button2 id = 13  onclick = chooseButton1(this.id)>13</button>"+
+  "<button type='button' class = select-button2 id = 14  onclick = chooseButton1(this.id)>14</button>"+
+  "<button type='button' class = select-button2 id = 15  onclick = chooseButton1(this.id)>15</button>"+
+  "<button type='button' class = select-button2 id = 16  onclick = chooseButton1(this.id)>16</button>"+
+  "<button type='button' class = select-button2 id = 17  onclick = chooseButton1(this.id)>17</button>"+
+  "<button type='button' class = select-button2 id = 18  onclick = chooseButton1(this.id)>18</button>"+
+  "<button type='button' class = select-button2 id = 19  onclick = chooseButton1(this.id)>19</button>"+
+  "<button type='button' class = select-button2 id = 20  onclick = chooseButton1(this.id)>20</button>"+
+  "<button type='button' class = select-button2 id = 21  onclick = chooseButton1(this.id)>21</button>"+
+  "<button type='button' class = select-button2 id = 22  onclick = chooseButton1(this.id)>22</button>"+
+  "<button type='button' class = select-button2 id = 23  onclick = chooseButton1(this.id)>23</button>"+
+  "<button type='button' class = select-button2 id = 24  onclick = chooseButton1(this.id)>24</button>"+
+  "<button type='button' class = select-button2 id = 25  onclick = chooseButton1(this.id)>25</button>"+
+  "<button type='button' class = select-button2 id = 26  onclick = chooseButton1(this.id)>26</button>"+
+  "<button type='button' class = select-button2 id = 27  onclick = chooseButton1(this.id)>27</button>"+
+  "<button type='button' class = select-button2 id = 28  onclick = chooseButton1(this.id)>28</button>"+
+  "<button type='button' class = select-button2 id = 29  onclick = chooseButton1(this.id)>29</button>"+
+  "<button type='button' class = select-button2 id = 30  onclick = chooseButton1(this.id)>30</button>"+
+  "<button type='button' class = select-button2 id = 31  onclick = chooseButton1(this.id)>31</button>"+
+  "<button type='button' class = select-button2 id = 32  onclick = chooseButton1(this.id)>32</button></div>"+
+  
```

Comment:
You might consider adding elements with Jquery or the like (eg a loop) to produce these elements, vs. hard coding them all.